### PR TITLE
Moved weight fields to StockPackage model #7530

### PR DESCRIPTION
**Pull request by PritishC**

### CODE REVIEW

File: shipment.py [hunk 1]
```diff
@@ -50,7 +43,48 @@ class ShipmentOut:
...
+    __name__ = 'stock.shipment.out'
+
+    tracking_number = fields.Char('Tracking Number', states=STATES)
+
+    is_international_shipping = fields.Function(
+        fields.Boolean("Is International Shipping"),
+        'on_change_with_is_international_shipping'
+    )
+
     override_weight = fields.Numeric("Override Weight", digits=(16,  2))
```
Comment (PritishC) [inline shipment.py]: Move to `Package`.

**Comment by PritishC** [comment]
Label generation wizard cannot have `override_weight` field now.

File: shipment.py [hunk 2]
```diff
@@ -24,16 +25,8 @@
...
-    tracking_number = fields.Char('Tracking Number', states=STATES)
-
-    is_international_shipping = fields.Function(
-        fields.Boolean("Is International Shipping"),
-        'on_change_with_is_international_shipping'
-    )
+class Package:
+    __name__ = 'stock.package'
 
     package_weight = fields.Function(
```
Comment (prakashpp) [inline shipment.py]: `weight`

**Comment by prakashpp** [comment]
Also add `CHANGELOG`

File: CHANGELOG [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Changelog
+=========
+
+New dependent modules:
+
+* stock_package
+
+Changes:
+
+* Shifted weight function fields from ShipmentOut to Package model #7530
```
Comment (tarunbhardwaj) [inline CHANGELOG]: refer commit id

**Pull request closed without merging**